docs/src/main/sphinx/sql/drop-role.md

```diff
@@ -3,7 +3,7 @@
 ## Synopsis
 
 ```text
-DROP ROLE role_name
+DROP ROLE [ IF EXISTS ] role_name
```

Comment:
Add docs for this in the description as well

Reply:
Done.